bullet_train-sortable/app/javascript/controllers/dragula-sortable_controller.js

```diff
@@ -0,0 +1,88 @@
+import { Controller } from "@hotwired/stimulus"
```

Comment:
This is the old `sortable_controller.js` just renamed to `dragula-sortable_controller.js` so that it can still be referenced by people who want to use it.